model/project_parser_test.go

```diff
@@ -1188,3 +1196,88 @@ tasks:
 	assert.Equal("somethingElse", proj.Loggers.Agent[1].Type)
 	assert.Equal("commandLogger", proj.Tasks[0].Commands[0].Loggers.System[0].Type)
 }
+
+func TestParserProjectPersists(t *testing.T) {
+	simpleYml := `
+loggers:
+  agent:
+    - type: something
+      splunk_token: idk
+    - type: somethingElse
+tasks:
+- name: task_1
+  depends_on:
+  - name: embedded_sdk_s3_put
+    variant: embedded-sdk-android-arm32
+  commands:
+  - command: myCommand
+    params:
+      env:
+        ${MY_KEY}: my-value
+        ${MY_NUMS}: [1,2,3]
+    loggers:
+      system:
+       - type: commandLogger
+functions:
+  run-make:
+    command: subprocess.exec
+    params:
+      working_dir: gopath/src/github.com/evergreen-ci/evergreen
+      binary: make
+      env:
+        CLIENT_URL: https://s3.amazonaws.com/mciuploads/evergreen/${task_id}/evergreen-ci/evergreen/clients/${goos}_${goarch}/evergreen
+`
+
+	for name, test := range map[string]func(t *testing.T){
```

Comment:
Have you tried running this test on mongodb/mongo and 10gen/mms?

Reply:
Yes I should have mentioned that, I did run this on both of their projects with no problems.

Reply:
should we write an integration test that does the "fetch all projects"? or at least open a ticket for that